modules/01 JavaScript Programming/Assignments/20/answerJS20.txt

```diff
@@ -0,0 +1,11 @@
+function myFilter(arr,callback)
+{
+    var result=[];
+    result=arr.filter(callback);
```

Comment:
If we're using "arr.filter" (the original "filter" function) we don't need to write a filter of our own.
Please re-write your solution without using the original "filter" and email it to me